### ISSUE TITLE
Add a faster way to pull out the drill from inventories

### ISSUE THREAD

**Issue by asumagic** [transhumandesign/kag-base#1696]
**Current in-game behaviour.**

When playing as a builder, you have to open the inventory, flick to wherever the drill is in your inventory, and start using it.

**Proposed Change**

It should be easier to pull out e.g. by tapping action3 (space). Having to flick through the menu adds a pointless/artificial mechanical constraint to how fast you can pull out the drill.

If this becomes a balance problem then something else to counter-balance would be better than not solving this issue at all.

**Additional context**

N/A

**Comment by TerminalHash** [comment]
I don't think it will affect the balance in any way, in any case further actions depend already on the player and his reaction.

**Comment by NoahTheLegend** [comment]
if this fixes that drills sometimes disappear clientside when you put them in the last slot im in favour

**Comment by asumagic** [comment]
> if this fixes that drills sometimes disappear clientside when you put them in the last slot im in favour

I think this is a separate engine-side issue, which IIRC is worsened by lag (or lagging behind ticks clientside?). I don't think it would directly solve it.

**Comment by asumagic** [comment]
Actually action3 would probably be unsuitable as it can be used to rotate blocks, so it would likely often happen that you rotate a block instead of pulling out the drill...
